### ISSUE TITLE
USB A port died when plugging in iPad

### ISSUE THREAD

**Issue by HenryLittle** [daliansky/XiaoMi-Pro#78]
I'm using the latest EFI(9-28), when I tried to plug in my iPad with the original cable(which works well on windows and my friend's macbook ). At first it would show that it's charging but after a few seconds it stops, then it's impossible to use the port (the iPad is also not recognized by iTunes) again without shutdown and restart.
I have tried to remove USBPower and replace it with the previous USBInjectAll and SSDT-USB.aml but the issue still exists, I wonder if there is any way that can fix it.

**Comment by stevezhengshiqi** [comment]
@HenryLittle Hi, I just tested with my iPad mini, and it works well. I suggest you can try to delete the following code by using `Text Editor.app`:
`<key>HighCurrent</key>
			<true/>`
Then tell me whether it works by doing this, thanks.

**Comment by HenryLittle** [comment]
I changed the key value to false and it works like charm, thanks a lot ;)

**Comment by stevezhengshiqi** [comment]
@HenryLittle Thank you for testing! I will change it in repository.

**Comment by stevezhengshiqi** [comment]
Changed.

**Comment by HenryLittle** [comment]
@stevezhengshiqi Hi, sorry to bother you again, but it seems that this fix is not stable, as I was using my iPad(pro 10.5) today the port just died again(with HighCurrent set to false). I also noticed that when I plug my android phone in it says charging rapidly, which won't happen on my another computer. It seems really wired.

**Comment by stevezhengshiqi** [comment]
Currently I am not able to fix this problem because I don't have this kind of issue on my laptop. Maybe you can test and try some USB Power injection in https://www.tonymacx86.com/threads/guide-usb-power-property-injection-for-sierra-and-later.222266/

**Comment by HenryLittle** [comment]
Thanks, I'm currently using the official use-c adapter with PD, and it works fine(at least for now). I will try the injections when I have time.

**Comment by MAX-POLKOVNIK** [comment]
@HenryLittle, @stevezhengshiqi Hello. I'm facing same problem. Today I tried to plug my workmate's iPad (not mini) and usb port died. It starts working only if I shutdown laptop. If I restart macOS or boot in Windows without shutdown it will not work. iPad had low battery charge. I guess this is a reason because few days ago I tried with almost fully charged and there are no problems with usb. I'll try it again after holidays to get more info.
